common/static/common/js/spec/components/feedback_spec.js

```diff
@@ -1,387 +1,383 @@
-// Generated by CoffeeScript 1.6.1
```

Comment:
@dan-f Add a query parameter of "w=1" to the URL to see what actually changed.

The JS translation of the CoffeeScript file had already been checked in, but it doesn't have much history due to the fact that the CoffeeScript change history itself was lost. As a separate commit, I changed this file to follow some of our JS guidelines (including indent size), and cleaned up some JSHint violations. I've been in this file a couple of times and have found the blatent disregard of readable JavaScript to be a problem. If you want me to pull this commit out of the JQuery PR, I can submit it separately.